analysis/trigger_threshold_tables.qmd

```diff
@@ -0,0 +1,243 @@
+---
+title: "Thresholds by LT table"
+format: html
+editor: visual
+---
+
+## Intro
+
+Code takes output from `_targets.R` pipeline to create nicely formatted tables with thresholds for sharing.
+
+```{r}
+library(targets)
+library(tidyverse)
+library(gghdx)
+library(gt)
+gghdx()
+
+tar_load(df_all_thresholds_rp4)
+
+ldf_insuv <-  df_all_thresholds_rp4 %>% 
+  filter(forecast_source=="INSUVIMEH") %>% 
+  split(.$window)
+
+ldf_ecmwf <-  df_all_thresholds_rp4 %>% 
+  filter(forecast_source=="ECMWF MARS") %>% 
+  # we keep ECMWF forecast for Guatemala only when LT == 0 (i.e in Primera)
+  filter(
+    !(adm0_es == "Guatemala" & 
+      lt %in% c(1:3))
+  ) %>% 
+  split(.$window)
```

Comment:
If you wanted to stay tidy verse, you can do `group_by(window) %>% group_split()`. This is cleaner, just noting for interest!

Reply:
I've liked the idea of `group_split()` in the past, but abandoned it a while ago as it removes the names from the list items which I don't like because I often think they are useful. There are various issues/threads about this on GH with some work around if i recall correctly, but I remember them not being as clean as just calling `split()` in the end.